### PR TITLE
Palak_Assignment_20_(30+10+10+10)Min

### PR DESCRIPTION
**Pull request by PalakSJun20**
For Program  missing number in integer array , can u help how to print missing integer Number as of now it is printing index value. 
image

image

image
image

### CODE REVIEW

File: src/palakSJun20/ArrayDuplicateElement.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+if an array with length 6 contains numbers {0, 3,
+1, 2, 3}, then duplicated number is 3.*/
+package palakSJun20;
+
+public class ArrayDuplicateElement {
+	//Method to find duplicate number on Integer Array
+	void findDuplicateNumberOnIntegerArray(int[] array) {
+		for (int i = 0; i < array.length; i++) {
+			for (int j = i + 1; j < array.length; j++) {
+				if (array[i] == array[j]) {
```
Comment (NikhilMJun20) [inline src/palakSJun20/ArrayDuplicateElement.java:13]: Perfect logic!

File: src/palakSJun20/ReverseIntegerNumbers.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+	//Method to Reverse the integer Numbers
+	void reverseInteger(int number) {
+		int reverse = 0;
+		int temp = number;
+		while (temp > 0) {
+			reverse = temp % 10;
+			temp = temp / 10;
+			System.out.print(reverse);
+		}
```
Comment (NikhilMJun20) [inline src/palakSJun20/ReverseIntegerNumbers.java:16]: The output is exactly as expected.
But, here integer is not reversed exactly - it is chopped digit by digit and printed in reverse order.
Please implement integer reverse logic - which is just a small extension to your logic.

File: src/palakSJun20/ArrayMissingInteger.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	
+	void findArrayMissingInteger(int[] array){
+		for (int i = 0; i < array.length; i++) {
+			int min = array[0];
+			int max = array[array.length - 1];
+			if (!(array[i] >= min && array[i] <= max)) {
+				System.out.println("Number is missing " + array[i]);
+			}
+		}
+	}
```
Comment (NikhilMJun20) [inline src/palakSJun20/ArrayMissingInteger.java:19]: This logic is incorrect. Output is incorrect.
There is simple and crisp logic available for finding missing number.
The i/p array has to be numbers from 1 to 10 with one number missing - hardcoded. You have taken different array. Please make the corrections.

**Review by NikhilMJun20 — CHANGES_REQUESTED** [review]
Please implement the suggestions.

File: src/palakSJun20/ReverseIntegerNumbers.java (hunk 2 above)
Comment (PalakSJun20) [reply src/palakSJun20/ReverseIntegerNumbers.java:16]: Changes Updated.

File: src/palakSJun20/ArrayMissingInteger.java (hunk 3 above)
Comment (PalakSJun20) [reply src/palakSJun20/ArrayMissingInteger.java:19]: Changes done

**Review by NikhilMJun20 — APPROVED** [review]

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**